pyproject.toml

```
@@ -0,0 +1,75 @@
+[project]
+name = "remote-server"
+version = "0.1.0"
+description = "NVDA Remote Access remote relay server."
+readme = "README.md"
+requires-python = ">=3.13.3"
+dependencies = [
+    "pyopenssl>=25.1.0",
+    "service-identity>=24.2.0",
+    "twisted>=24.11.0",
+]
```

I think we should at the very least, pin to the minor version if not the full version.
Otherwise if a new server spins up, and installs the latest version of dependencies, the server could break.


```suggestion
dependencies = [
    "pyopenssl~=25.1",
    "service-identity~=24.2",
    "twisted~=24.11",
]
```